regression/cbmc-library/memcpy-06/test.desc

```diff
@@ -0,0 +1,10 @@
+CORE
+main.c
+
+function 'memcpy' is not declared
+parameter "memcpy::dst" type mismatch
```

Comment:
⛏️ mixture of " and '

Reply:
It seems there's some more work after #4875 left to be done. I'll create a separate PR for this.